libraries/splunk_password.rb

```diff
@@ -13,6 +13,7 @@ module CernerSplunk
   def self.splunk_encrypt_password(plain_text, splunk_secret, xor = true)
     # Prevent double encrypting values
     return plain_text if plain_text.start_with? '$1$'
+    return plain_text if plain_text.start_with? '$7$'
```

Comment:
It looks like start_with takes a list of arguments.  So this could be:
`return plain_text if plain_text.start_with? '$1$', '$7$'`

Reply:
Changed over in https://github.com/cerner/cerner_splunk/pull/237/commits/9b2c26d1e6f9bb5e4a7e9e7c711cdae6dfe3424f